tests/acceptance/apps_dns/apps_dns_suite_test.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2020 SUSE
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apps_dns_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/kubernetes-sigs/minibroker/pkg/kubernetes"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/SUSE/kubecf-apps-dns/tests/acceptance/apps_dns/fake"
+)
+
+var (
+	sdc           *fake.ServiceDiscoveryController
+	dnsAddr       string
+	serveShutdown chan<- struct{}
+	serveErrCh    <-chan error
```

Comment:
Why does one end in `Ch` and the other doesn't? Should we be consistent here?

Reply:
No idea. Fixed in 26cb11f.